src/brucke_subscriber.erl

```diff
@@ -168,7 +168,7 @@ do_handle_message_set(#{ route := Route
   #route{ upstream = {_UpstreamClientId, Topic}
         , downstream = {DownstreamClientId, DownstreamTopic}
         } = Route,
-  PartCnt = brod_client:get_partitions_count(DownstreamClientId, DownstreamTopic),
+  {ok, PartCnt} = brod_client:get_partitions_count(DownstreamClientId, DownstreamTopic),
```

Comment:
Looks like dialyzer material. Add ```make dialyze``` to travis.yml?

Reply:
yes!